lib/internal/Magento/Framework/View/Helper/SecureHtmlRenderer.php

```diff
@@ -119,8 +119,10 @@ function {$listenerFunction} () {
                         targetElement = event.target;
                     }
                     {$listenerFunction}.apply(targetElement);
-                }
+		    event.preventDefault();
```

Comment:
Please, make sure that the indentation is correct for all lines of code after your fix